tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/Intersections.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using static System.Runtime.Intrinsics.X86.Avx;
+using static System.Runtime.Intrinsics.X86.Avx2;
+using System.Runtime.Intrinsics.X86;
+using System.Runtime.Intrinsics;
+using System.Runtime.CompilerServices;
+using System;
+
+internal struct Intersections
+{
+    public Vector256<float> Distances;
+    public Vector256<int> ThingIndeces;
+
+    public static readonly Vector256<float> NullDistance = SetAllVector256<float>(float.MaxValue);
```

Comment:
Why `float.MaxValue`, instead of something like `NaN` or `Infinity`?

Reply:
Because we need to compare the `NullDistance` against the result of `Intersect` in method `MinIntersect`.